### PR TITLE
Added loader in Captcha

### PR DESCRIPTION
**Pull request by YAS-SIIN**

### CODE REVIEW

File: src/containers/HcaptchaContainer.tsx [hunk 1]
```diff
@@ -170,15 +175,19 @@ export default function HcaptchaContainer({ navigation }: { navigation: Props['n
                                     <View style={{ flexDirection: 'row', alignItems: 'center' }}>
                                         <Checkbox.Android
                                             status={code ? 'checked' : 'unchecked'}
-                                            onPress={() => {
+                                            onPress={() => { 
+                                                setSuccess(!success);
```
Comment (theblockstalk) [inline src/containers/HcaptchaContainer.tsx:179]: Better is `setSuccess(success => !success)` because useState hooks are async

**Review by theblockstalk — COMMENTED** [review]
does this also remove the grey screen? that is the bigger issue

File: src/containers/HcaptchaContainer.tsx (hunk 1 above)
Comment (YAS-SIIN) [reply src/containers/HcaptchaContainer.tsx:179]: Done.

Comment (YAS-SIIN) [reply src/containers/HcaptchaContainer.tsx:179]: let me find other solution about the gray screen

**Review by theblockstalk — APPROVED** [review]

**Pull request merged**